.github/workflows/doc-index-updater-branch.yaml

```diff
@@ -21,6 +21,10 @@ jobs:
       - name: Clone Repo
         uses: actions/checkout@v2
 
+      - name: Test for RC release
+        working-directory: ./medicines/doc-index-updater
+        run: cmd=`cat Cargo.toml | grep mhra_products_search_client | sed -e 's/"//' | sed -e 's/"//'` && if [[ $cmd == *"-rc"* ]];then echo rc dependencies not allowed for this package; exit 1; fi
```

Comment:
can be simplified to:-
`case $(grep <Cargo.toml mhra_products_search_client) in *"-rc"*) echo "Only stable versions of mhra_products_search_client should be used"; exit 1 ;; esac`